src/docker/docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: '3.7'
+version: "3.7"
```

Comment:
Prettier.

But why `"` while above it was `'`...

Reply:
Hmm a bit weird, but fine for me to merge.